src/views/Migrate.vue

```diff
@@ -19,28 +41,45 @@ import { Component, Vue } from 'vue-property-decorator';
 import { AccountInfo } from '@/lib/AccountInfo';
 import { WalletStore } from '@/lib/WalletStore';
 import { WalletInfo, WalletType } from '@/lib/WalletInfo';
-import { SmallPage } from '@nimiq/vue-components';
+import { SmallPage, PageHeader, PageBody, PageFooter, AccountList } from '@nimiq/vue-components';
 import Network from '@/components/Network.vue';
 import Loader from '@/components/Loader.vue';
-import { KeyguardClient } from '@nimiq/keyguard-client';
+import KeyguardClient from '@nimiq/keyguard-client';
```

Comment:
```suggestion
import { ExportRequest, LegacyKeyInfoObject } from '@nimiq/keyguard-client';
```

Reply:
My problem with this is, that `ExportRequest` is not unique, because the Hub also has this type itself. Thus I would have to import it `as KeyguardExportRequest`, which then makes no difference anymore, I think.

Reply:
In general it does make a difference for tree-shake-ability. Importing a whole name space is not good practice and as far as I know will result in all imports being bundled.
Here in the case of the KeyguardClient it doesn't matter.